focus-within.js

```diff
@@ -43,6 +43,30 @@ export const focusWithin = (baseElement) => class extends baseElement {
     };
   }
 
+  /**
+   * When current focused element is detached, removes focus.
+   */
+  set _currentFocusedElement(val) {
+    const oldValue = this.__currentFocusedElement;
```

Comment:
@nirmalbaldaniya-dream I noticed that this behavior is required only for iOS, while we have implemented it for all the browsers. So, for the performance reason, we need to update it to be applied only for the iOS